src/utils/typography.js

```diff
@@ -10,119 +10,138 @@ export const primaryFont = "tondo, sans-serif";
 //     button: '1.35em'
 // };
 
-
 export const H1 = styled.h1`
-    font-family: ${primaryFont};
-    font-size: 1.6em;
-    line-height: 1.3em;
-    letter-spacing: -0.017em;
-    color: ${props => props.color ? props.color : colors.primaryGray};
-    margin: ${props => props.margin ? props.margin : 0};
+  font-family: ${primaryFont};
+  font-size: 1.6em;
+  line-height: 1.3em;
+  letter-spacing: -0.017em;
+  color: ${(props) => (props.color ? props.color : colors.primaryGray)};
+  margin: ${(props) => (props.margin ? props.margin : 0)};
 `;
 
 export const H2 = styled.h2`
-    font-family: ${primaryFont};
-    font-size: 1.25em;
-    line-height: 1.05em;
-    letter-spacing: -0.018em;
-    font-weight: ${props => {
-        switch (props.wight) {
-            case "light":
-                return "100"
-            case "regular":
-                return "400"
-            case "bold":
-                return "700"
-            default:
-                return "400"
-        }
-    }};
-    color: ${props => props.color ? props.color : colors.primaryGray};
-    margin: ${props => props.margin ? props.margin : 0};
+  font-family: ${primaryFont};
+  font-size: 1.25em;
+  line-height: 1.05em;
+  letter-spacing: -0.018em;
+  font-weight: ${(props) => {
+    switch (props.wight) {
+      case "light":
+        return "100";
+      case "regular":
+        return "400";
+      case "bold":
+        return "700";
+      default:
+        return "400";
+    }
+  }};
+  color: ${(props) => (props.color ? props.color : colors.primaryGray)};
```

Comment:
I think it's better to create variants than to use props for colors.

Reply:
I. did create color variables to use as the props for different things

Reply:
I think it's better when it come to design system to only use constraints and modifier: 
This h1 could be used with those variations ex:

<Typography.h1 modifiers={"primary"}> Hello</Typography.h1>

This will prevent someone to write a wrong color when creating the component.

You can use modifiers. https://github.com/Decisiv/styled-components-modifiers